interface_specifications/pe_openapi/apis/v1/e1_pe_presentation_statuses_put.md

```diff
@@ -0,0 +1,30 @@
+### Status Notification to Holder
+
+`PUT <holder-example.io>pe/presentations/{presentation_id}/statuses`
+
+
+#### Description
+
+This is an optional interaction and hence party can choose not to implement it. The PE REST Backend invokes an HTTP REST API exposed by the party to notify about the status of exchange.
+
+#### Parameters
+
+Path Variables:
+* `presentation_id`: presentation id e.g. `a30e3b91-fb77-4d22-95fa-871689c322e2`
+
+Request Body:
+```json
+{
+  "thread":{
+    "id": "f1ca8245-ab2d-4d9c-8d7d-94bf310314ef"
+  },
+  "presentation_id": "a30e3b91-fb77-4d22-95fa-871689c322e2",
```

Comment:
presentation_id is already in the path so no need to have it in the request body

Reply:
We need this presentation id to be present in the object to validate it.
We can indeed copy the id present in the path to the object on the backend side, in controller or filters.

So, shall we remove the redundancy ?

Reply:
removing from request body